platform-tests/src/test/java/org/eclipse/deeplearning4j/frameworkimport/tensorflow/TFGraphTestAllHelper.java

```diff
@@ -123,6 +123,7 @@ public ModelLoadResult apply(File file, String name) {
                 SameDiff result = tensorflowFrameworkImporter.runImport(file.getAbsolutePath(), dynamicVariables, suggestDynamicVariables);
                 return new ModelLoadResult(result, graphDef);
             }catch(Exception e) {
+                e.printStackTrace();
```

Comment:
Wouldn't it be better to log this?

Or is this a debugging leftover?

Reply:
Why not make it a loud error instead of printing to the console?

Reply:
Done.